src/Telegram.Bot/Requests/Available methods/Commands/DeleteMyCommandsRequest.cs

```diff
@@ -6,9 +6,9 @@
 namespace Telegram.Bot.Requests;
 
 /// <summary>
-/// Use this method to delete the list of the bot's commands for the given
+/// Use this method to delete the list of the bot`s commands for the given
```

Comment:
` (backtick) is grammatically incorrect. it should be ' (tick) or ’ (apostrophe)